assets/app/scripts/controllers/build.js

```diff
@@ -70,6 +71,21 @@ angular.module('openshiftConsole')
               }
               $scope.build = build;
             }));
+            watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
+              if (action === "DELETED") {
+                $scope.alerts["deleted"] = {
+                  type: "warning",
+                  message: "Build configuration " + $scope.buildConfigName + " has been deleted."
+                };
+              }
+              $scope.buildConfig = buildConfig;
+              if ($scope.buildConfig.metadata.annotations && $scope.buildConfig.metadata.annotations["openshift.io/build-config.paused"] === "true") {
+                $scope.alerts["paused"] = {
+                  type: "warning",
+                  message: "Building from build configuration " + buildConfig.metadata.name + " has been paused."
+                };
+              }
```

Comment:
Looks like the comment about removing the warning when the annotation removed is still valid.